api/catalog/api/utils/pagination.py

```diff
@@ -1,48 +1,18 @@
 from rest_framework.pagination import PageNumberPagination
 from rest_framework.response import Response
 
-from catalog.api.utils.exceptions import get_api_exception
-
 
 class StandardPagination(PageNumberPagination):
-    page_size_query_param = "page_size"
-    page_query_param = "page"
+    page_size_query_param = None
```

Comment:
If we are not going to use these, then we can simply remove the `page_size_query_param`. It is already set to to `None` in the parent class.

Reply:
And this is the only minor thing blocking this PR, I guess.

Reply:
Ah this is a good catch, I saw that `page_query_param` was `'page'` but I missed that `page_size_query_param` was `None`.

Reply:
Fixed in c461450.